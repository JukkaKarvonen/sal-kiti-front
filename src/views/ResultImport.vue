<template>
  <div class="import">
    <Competition />
    <ResultImport />
  </div>
</template>

<script>
/**
 * Result import view
 */
import Competition from "@/components/CompetitionInformation.vue";
import ResultImport from "@/components/ResultImport.vue";

export default {
  name: "ResultImportView",
  components: {
    Competition,
    ResultImport
  },
  created() {
    document.title =
      this.$t("title.result_import") + " | " + this.$t("title.prefix");
  }
};
</script>
