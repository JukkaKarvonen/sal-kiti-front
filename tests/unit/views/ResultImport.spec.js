import { shallowMount, createLocalVue } from "@vue/test-utils";
import ResultImport from "@/views/ResultImport.vue";

import BootstrapVue from "bootstrap-vue";

const localVue = createLocalVue();
localVue.use(BootstrapVue);

describe("ResultImport.vue", () => {
  it("renders content", () => {
    const wrapper = shallowMount(ResultImport, {
      localVue
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
