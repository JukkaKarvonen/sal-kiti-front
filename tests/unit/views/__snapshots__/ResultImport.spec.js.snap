// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultImport.vue renders content 1`] = `
<div
  class="import"
>
  <competition-stub />
   
  <resultimport-stub />
</div>
`;
